### DISCUSSION TITLE
The M2 of the drone is always on

### DISCUSSION THREAD

**Discussion by zyxzxy** [bitcraze/discussions#1640] [Q&A]
hi，
In order to solve https://github.com/orgs/bitcraze/discussions/1633, I tried to flash the firmware using the client, but after the flashing was completed, the M2 of the drone started to light up constantly. This is very similar to https://github.com/orgs/bitcraze/discussions/1384, but my drone can't connect to the bootloader, so it can't be solved like him, what should I do now?
![Screenshot from 2024-11-08 09-34-56](https://github.com/user-attachments/assets/f5c92b96-1fb7-4a17-9a75-58002b3b3958)

**Comment by Bananazooka** [comment]
Hi, it seems your Crazyflie is bricked. You should try the steps described [here](https://www.bitcraze.io/support/troubleshooting/#crazyflie-is-bricked).

**Reply by zyxzxy** [reply]
Thank you for your reply. Yes, I have tried this. You can take a look at the video I sent. Is there anything wrong with my operation? Why can't I connect to the client?

**Reply by Bananazooka** [reply]
Hi, it seems your problem is solved here #1633.
